data/state/accountsDB.go

```diff
@@ -77,6 +77,8 @@ type AccountsDB struct {
 	numCheckpoints       uint32
 	loadCodeMeasurements *loadingMeasurements
 	stopWatch            *core.StopWatch
+	ctx                  context.Context
```

Comment:
can we not store ctx and simply pass it?

Reply:
Refactored.